internal/dnfjson/dnfjson.go

```diff
@@ -113,6 +113,30 @@ func ReposFromRPMMD(rpmRepos []rpmmd.RepoConfig, arch string, releaseVer string)
 	return dnfRepos, nil
 }
 
+func DepsToRPMMD(dependencies []PackageSpec, repos []rpmmd.RepoConfig) []rpmmd.PackageSpec {
```

Comment:
Is this meant as a transitional measure until `rpmmd` has been replaced, or will both stay around?

Reply:
This is necessary because the returned data from `dnf-json` needs to be augmented with repository information (we set `CheckGPG` on each package based on the repository configuration).
We could move this into the `dnf-json` script.

This conversion [also happens in `main`](https://github.com/osbuild/osbuild-composer/blob/2c2668f4933129bbedf20bc68fb8b727799a447c/internal/rpmmd/repository.go#L495-L513).

Reply:
Or move it to be internal to the `dnfjson` package, but not into the `dnf-json` script. I mean that the package objects you return from `Depsolve()` and friends would have the right repo pointers etc.

Reply:
Indeed, it doesn't need to be public.  Lot of things in `dnfjson` don't need to be actually.